src/core/lib/surface/call.c

```diff
@@ -1467,7 +1444,6 @@ static void receiving_stream_ready(grpc_exec_ctx *exec_ctx, void *bctlp,
       grpc_byte_stream_destroy(exec_ctx, call->receiving_stream);
       call->receiving_stream = NULL;
     }
-    add_batch_error(exec_ctx, bctl, GRPC_ERROR_REF(error), true);
```

Comment:
I think this is safe to remove.  The only places it is called from is ```receiving_stream_ready_in_call_combiner``` and ```receiving_initial_metadata_ready```.  In both cases we already call ```add_batch_error``` with this error.